client/flutter/lib/components/page_scaffold.dart

```diff
@@ -7,34 +7,37 @@ class PageScaffold extends StatelessWidget {
 
   final List<Widget> body;
 
-  final BuildContext context;
   final EdgeInsets padding;
   final bool showBackButton;
 
-  PageScaffold(this.context,
-      {@required this.body,
-      @required this.title,
-      this.subtitle = "WHO Coronavirus App",
-      this.padding = EdgeInsets.zero,
-      this.showBackButton = true});
+  PageScaffold({
+    @required this.body,
+    @required this.title,
+    this.subtitle = "WHO Coronavirus App",
+    this.padding = EdgeInsets.zero,
+    this.showBackButton = true,
+  });
 
   @override
   Widget build(BuildContext context) {
-    return Material(
-        color: Colors.grey.shade200,
-        child: Padding(
+    return Scaffold(
```

Comment:
The `PageScaffold` now actually uses/is a `Scaffold`. This way `SnackBar` can be shown.